resources/codegen_inputs/mvar.rs

```diff
@@ -0,0 +1,31 @@
+#![parse_module(read_fonts::tables::mvar)]
+
+/// The [MVAR (Metrics Variations)](https://docs.microsoft.com/en-us/typography/opentype/spec/mvar) table
+table Mvar {
+    /// Major version number of the horizontal metrics variations table — set to 1.
+    /// Minor version number of the horizontal metrics variations table — set to 0.
+    version: MajorMinor,
+    /// Not used; set to 0.
+    #[skip_getter]
+    _reserved: u16,
+    /// The size in bytes of each value record — must be greater than zero.
+    value_record_size: u16,
```

Comment:
so we probably want to support this, although I'm okay if we do it later. I have a pattern that handles this that I'm using in STAT, and it should be portable to here.

Reply:
I'm really leery of adding complexity for this kind of thing and I suspect (like fvar's VariationAxisRecord) that this size is forever fixed because implementations have hard coded it. Happy to revisit in the future.